{"status":"OK","subtests":[{"name":"When in a non-secure context, getCurrentPosition()'s errorCallback is asynchronously called.","status":"FAIL"},{"name":"When in a non-secure context, watchPosition()'s errorCallback is asynchronously called.","status":"FAIL"},{"name":"When in a non-secure context, the getCurrentPosition() errorCallBack gets a GeolocationPositionError with the correct error code.","status":"FAIL"},{"name":"When in a non-secure context, the watchPosition() errorCallBack gets a GeolocationPositionError with the correct error code.","status":"FAIL"}]}